webrtc.html

```diff
@@ -481,8 +481,9 @@
           </li>
 
           <li>
-            <p>Initialize an internal variable to represent a queue of
-            <code>operations</code> with an empty set.</p>
+            <p>On the <var>connection</var>, reserve an internal slot
+            <var>operations</var>, and initialize an internal slot
+            <var>operationsLength</var> to <code>0</code>.</p>
           </li>
```

Comment:
We [seem to be using](http://w3c.github.io/webrtc-pc/#rtccertificate-interface)  "internal slot" now, instead of "internal variable". Open to discuss.

The conceptual difference I think you're pointing at is "chain of operations" vs. "queue of operations". Established vernacular seems to be that promise functions [chain](http://www.html5rocks.com/en/tutorials/es6/promises/#toc-chaining), they don't [queue](http://www.w3.org/TR/html5/webappapis.html#task-queue).